update-engine/src/display/group_display.rs

```diff
@@ -236,18 +236,9 @@ impl GroupDisplayStats {
     }
 
     fn apply_result(&mut self, result: AddEventReportResult) {
-        // Process result.after first to avoid integer underflow.
```

Comment:
This wasn't really good enough to avoid integer underflow 😅, so I decided to just check for equality and then do the more obvious thing of applying `before` before `after`.